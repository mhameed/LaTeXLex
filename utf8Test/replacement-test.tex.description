Date: 10/07/02
Added by: EHC
Description: the full LaTeXLex should be run on replacement test to check that the utf8 replacement is functioning as expected. At the moment this tests only the Greek and so the test should be expanded.  