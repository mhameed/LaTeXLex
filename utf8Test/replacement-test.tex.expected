α β γ δ ϵ ζ η θ ι κ λ μ ν ξ ο π ρ σ τ υ ϕ χ ψ ω 
